rasgoql/rasgoql/data/base.py

```diff
@@ -1,12 +1,17 @@
 """
 Base DataWarehouse classes
 """
+from __future__ import annotations
 from abc import ABC
 import re
-from typing import Union
+import os
+from typing import Union, Optional
+from collections import namedtuple
 
 import pandas as pd
 
+FQTN = namedtuple("FQTN", ["database", "schema", "table"], defaults=(None, None, None))
```

Comment:
This is basically just a pretty skin right now (works same as regular tuple so no usage change), but I want to use it to lead into a larger conversation of the usage throughout RasgoQL between using `fqtn` and `schema.table` in many functions that seem overkill to provide all that each time. 

Many data warehouses don't support switching databases, and I would imagine most use cases do not often switch between schemas. Would there be value add to have database and default_schema set at the class level, and by default only expect a table name in most functions, then have separate keyword arguments for schema, for example [how pandas does it with `to_sql`](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_sql.html#pandas.DataFrame.to_sql)